Assets/Scripts/NursePatrol.cs

```diff
@@ -5,8 +5,9 @@
 
 public class NursePatrol : MonoBehaviour
 {
-    public Transform[] points;
+    public GameObject Patrol_Waypoints;
```

Comment:
I think this can just be type `Transform`.

Reply:
Fixed